### PR TITLE
Border0launch

### PR DESCRIPTION
**Pull request by atoonk**
This PR introduced ```border0 run -- <command>``` 
specifically for Docker containers. And will provide the container with an HTTP and SSH socket for easy management access and an HTTP socket with information about the container.

For now, it works with Default policies only.  

Example docker file:
```
FROM nginx

COPY border0 /
RUN mkdir -p /root/.border0
COPY token /root/.border0/token

CMD ["/border0", "run", "--", "nginx", "-g", "daemon off;"]
```

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+			if err != nil {
+				log.Printf("error: %v", err)
+			}
+		}
+		// And let our parent know we're done with the clean up
+		cleanupDone <- true
+
+	}()
+
+	i := 0
```
Comment (btoonk) [inline cmd/run.go:142]: Can we use github.com/cenkalti/backoff/v4 here?

File: cmd/run.go [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+				log.Printf("error: %v", err)
+			}
+		}
+		// And let our parent know we're done with the clean up
+		cleanupDone <- true
+
+	}()
+	// Start a goroutine to wait for the process to exit
+
+	i := 0
```
Comment (btoonk) [inline cmd/run.go:226]: same here, can we use github.com/cenkalti/backoff/v4 here??

File: cmd/run.go (hunk 1 above)
Comment (atoonk) [reply cmd/run.go]: ack, done

File: cmd/run.go (hunk 2 above)
Comment (atoonk) [reply cmd/run.go]: ack, done

**Review by btoonk — APPROVED** [review]

**Pull request merged**